dissect/target/volumes/vmfs.py

```diff
@@ -34,25 +33,18 @@ def open_all(cls, volumes: list[BinaryIO]) -> Iterator[LogicalVolumeSystem]:
                 continue
 
     @staticmethod
-    def detect(fh: BinaryIO) -> bool:
+    def _detect(fh: BinaryIO) -> bool:
         vols = [fh] if not isinstance(fh, list) else fh
```

Comment:
This listifying does not really make sense (anymore), as the base class `detect()` only allows a single `fh` to be specified.